governance.md

```diff
@@ -1,133 +1,229 @@
+* Sponsored by 1 Approver or TOC member.
+  * With no objections from other Approvers
+  * Done through a PR to update the [membership.yml](membership.yml) file
+* May either self-nominated or be nominated by an Approver of the project
+
+#### Responsibilities and Privileges
+
+The following apply to the part of the codebase for which one would be a Reviewer.
+
+* Reviewer status may be a precondition to accepting large code contributions.
+* Responsible for project quality control via code reviews.
+  * Focus on code quality and correctness, including testing and factoring.
+  * May also review for more holistic issues, but not a requirement.
+* Expected to be responsive to reviews requested.
+* Assigned PRs to review related to project of expertise.
+  * Reviewers can self-assign into automatic reviewership by opening PRs against the `CODEOWNERS` file of the projects.
 
 ### Approvers
 
-Duties
+Approvers are able to both review and approve code contributions. While code review is focused on code quality and correctness, approval is focused on holistic acceptance of a contribution: backwards / forwards compatibility, adhering to conventions, performance and correctness issues, interactions with other parts of the system, and so-on.
+
+Approver status is scoped to a part of the codebase.
+
+#### Requirements
 
-* Final review for PRs
-* Merge code into master
-* All Reviewer duties
+* Reviewer of the codebase for at least 3 months.
+* Reviewed or merged at least 20 PRs to the codebase.
+* Nominated by at least 2 other Approvers, SIG Leads, or TOC members.
+  * With no objections from other Approvers or SIG Leads.
+  * Done through PR to update the [membership.yml](membership.yml) file.
 
-Qualification guidelines
+#### Responsibilities and Privileges
 
-* Is already fulfilling the Reviewer role
-* Belongs to an organization with significant stake in project
-* Known active participant in the community
-* Has submitted at least 5 PRs of significant scope that are merged
-* Has reviewed 5 PRs of significant scope
-* Sponsored by 2 Approvers
+* Approver status may be a precondition to accepting large code contributions.
+* Demonstrate sound judgment.
+* Responsible for project quality control via code reviews.
+  * Focus on holistic acceptance of a contribution, such as dependencies with other features, backwards / forwards compatibility, API definitions, etc.
+* Expected to be responsive to review requests.
+* Mentor contributors and Reviewers.
+* May approve code contributions for acceptance.
 
-[Current Approvers](https://github.com/spinnaker/community/blob/master/membership.yml)
+#### Expedited Approvership
+
+In some specific cases, Members can directly climb to Approver. In these cases, an issue must be made in the governance repository and voted on unanimously by the Technical Oversight Committee. Scenarios in which an Approver role can be expedited are:
+
+* Subproject ownership transfers.
```

Comment:
...what does this mean? I suppose you're specifically trying to call out the `spin` case here? It kinda felt like a one-off case, and maybe it's just enough to say it's up to the TOC's discretion in rare circumstances and with their unanimous approval?